hash-store/iam_policy_documents.tf

```diff
@@ -0,0 +1,68 @@
+data "aws_iam_policy_document" "read_policy" {
+  # This is based on the AmazonDynamoDBReadOnlyAccess
```

Comment:
Is this sentence missing a last word?

Reply:
I suppose it's inferred from the type of thing this is. Added the word policy.
https://github.com/wellcometrust/terraform-modules/blob/master/vhs/modules/vhs/iam_policy_document.tf#L2